scripts/setup_dev.py

```diff
@@ -52,7 +52,7 @@ def main():
     print("1. Run tests: uv run pytest")
     print("2. Format code: uv run black edam_mcp/")
     print("3. Start the server: uv run python -m edam_mcp.main")
-    print("4. Run examples: uv run python examples/basic_usage.py")
+    print("4. Run examples: uv run python examples/basic_usage_mapper.py")
```

Comment:
I think we should also add the suggester examples here
```suggestion
    print("4. Run examples: uv run python examples/basic_usage_mapper.py")
    print("5. Run examples: uv run python examples/basic_usage_suggester.py")
```